frontend/src/Commons/Forms/InputModules/FarmInformation/FarmInformation.tsx

```diff
@@ -103,7 +106,8 @@ const FarmInfoComponent: FC<InputModuleProps> = ({
                 id="FarmRegion"
                 label="Region"
                 options={options}
-                width="50%"
+                desktopWidth="468px"
+                mobileWidth="259px"
```

Comment:
Is this a good idea?

Reply:
I noticed the intention is to precisely follow the mockups, we should watch out though because this will me hell to refactor in the future. We're making our styling lesse flexible with a lot of hardcoding. Perhaps using constants could be helpful here... I'll leave it up to you though since you're the one who's dealing with it. Do whatever you think is and will be easier.

Reply:
Not the best, it's a quick and dirty response to making exact measurements of the mockup. if I use width, then I'll have conflicting decision on where I need to use the prop for. Either mobile, or desktop. If desktop widths are all the same, then I would probably have used percentage based on its parent container. However, it is not all the same. Which is why creating a fixed values worked for me to deliver what the designer has been envisioning.